src/App.jsx

```diff
@@ -1,7 +1,35 @@
 import "./App.css";
+import { useState } from "react";
+import Header from "./components/Header.jsx";
+import CategoryFilter from "./components/CategoryFilter.jsx";
+import RestaurantList from "./components/RestaurantList.jsx";
+import AddRestaurantModal from "./components/AddRestaurantModal.jsx";
+import RestaurantDetailModal from "./components/RestaurantDetailModal.jsx";
+import restaurantsData from "./data/restaurantsData.js";
```

Comment:
```suggestion
import Header from "./components/Header";
import CategoryFilter from "./components/CategoryFilter";
import RestaurantList from "./components/RestaurantList";
import AddRestaurantModal from "./components/AddRestaurantModal";
import RestaurantDetailModal from "./components/RestaurantDetailModal";
import restaurantsData from "./data/restaurantsData";
```
확장자를 명시하지 않으셔도 괜찮답니다😊 아래의 링크를 참고해 보세요!

- https://vite.dev/config/shared-options#resolve-extensions